akka-projection-cassandra/src/test/java/akka/projection/cassandra/CassandraProjectionTest.java

```diff
@@ -45,8 +45,8 @@
 
   @BeforeClass
   public static void beforeAll() throws Exception {
-    session = CassandraSessionRegistry.get(testKit.system()).sessionFor("akka.projection.cassandra");
-    offsetStore = new CassandraOffsetStore(session.delegate(), testKit.system().executionContext());
+    offsetStore = new CassandraOffsetStore(testKit.system());
+    session = new CassandraSession(offsetStore.session());
```

Comment:
Creating a new `javadsl. CassandraSession` like that is somewhat strange, not really intended user api to do it like that. I think it's better to grab it from CassandraSessionRegistry here

Reply:
This is a consequence of moving the session to inside of the CassandraOffsetStore (there was a TODO for that in the code), but it's also related to #154. 

I will revert it, but it was not my intention that users would access it like that. I should have make it private to make it clear.

Reply:
Anyway, I agree that having a an accessor to session in offset store just be able to reuse it in our tests is not a great idea. It's removed now.